gradle/libs.versions.toml

```diff
@@ -65,9 +66,11 @@ androidx-datastore-preferences = { module = "androidx.datastore:datastore-prefer
 androidx-fragment-ktx = "androidx.fragment:fragment-ktx:1.8.6"
 androidx-hilt-compiler = { module = "androidx.hilt:hilt-compiler", version.ref = "androidx-hilt" }
 androidx-hilt-work = { module = "androidx.hilt:hilt-work", version.ref = "androidx-hilt" }
+androidx-hilt-navigation-compose = { module = "androidx.hilt:hilt-navigation-compose", version.ref = "androidx-hilt" }
 androidx-lifecycle-livedata-ktx = { module = "androidx.lifecycle:lifecycle-livedata-ktx", version.ref = "androidx-lifecycle" }
 androidx-lifecycle-runtime-testing = { module = "androidx.lifecycle:lifecycle-runtime-testing", version.ref = "androidx-lifecycle" }
 androidx-lifecycle-viewmodel-compose = { module = "androidx.lifecycle:lifecycle-viewmodel-compose", version.ref = "androidx-lifecycle" }
+androidx-material3-window-size-class1-android = { group = "androidx.compose.material3", name = "material3-window-size-class-android", version.ref = "material3WindowSizeClassAndroid" }
```

Comment:
I'd like to share the version ref with the `androidx-compose-material3` dependency already defined.

Also, I haven't checked, but this isn't included transitively when the compose material3 dependency is imported?

Reply:
also, you don't need the `-android` part of the library name, the material3 compose libraries are Kotlin Multiplatform modules. Using `-android` just bypasses the multiplatform resolution to get the android platform version only. It's valid, but could be potentially confusing because everywhere that google references the library they don't include the platform modifier.

Reply:
and name the version catalog entry: `androidx-compose-material3-windowsize`

Reply:
```suggestion
androidx-compose-material3-windowsizeclass = { group = "androidx.compose.material3", name = "material3-window-size-class", version.ref = "compose-material3" }
```